modules/govuk/manifests/apps/signon.pp

```diff
@@ -37,7 +37,7 @@
     port               => $port,
     vhost_ssl_only     => true,
     health_check_path  => '/users/sign_in',
-    log_format_is_json => true,
+    legacy_logging     => false,
     vhost_aliases      => ['signonotron'],
     logstream          => $ensure_logstream,
```

Comment:
For reference, this param (set in hieradata) was [the one @bradwright was referring to](https://github.com/alphagov/signonotron2/pull/464#issuecomment-177001002). Do we want to set it to true?